11-multi-track/bb_views_part1/readme.md

```diff
@@ -87,7 +94,7 @@ approach for our collection views later.
 Define a view in `js/backbone/views/grumble.js`. You should define the view such
```

Comment:
Show how to implement `tagName` and `className` in `The el in Backbone` section.